ibvl/urls.py

```diff
@@ -34,21 +34,23 @@
     path('admin/', admin.site.urls),
     path('tracking/', access.tracking_dashboard, name='tracking_dashboard'),
     path('accounts/profile/', access.profile_view_redirect, name='profile'),
-    path('api/', include(api_urls))
+    path('api/', include(api_urls)),
 ]
 
 def redirect_to_login(request):
-    response = redirect('/accounts/login/')
+    response = redirect('accounts/login/')
     return response
 
-if settings.IS_DEVELOPMENT:
-    urlpatterns = urlpatterns + [
-        path('', backend_home_page, name='backend_home_page'),
-        path('accounts/logout', access.logout_view, name='logout')
-    ]
-if not settings.IS_DEVELOPMENT or os.getenv("AUTH_TENANT_ID", False):
-    urlpatterns = urlpatterns + [
+if os.getenv("AUTH_AZUREAD", False).lower() == 'true':
```

Comment:
Can this change from False to 'False' so that .lower() call does not produce an error?